### PR TITLE
Update simple_animal.dm

### PR DESCRIPTION
**Pull request by UndeadNappist**
Maximum cows cannot be exceeded.
Mind giving this a look over? @rhicora

### CODE REVIEW

File: code/modules/mob/living/simple_animal/simple_animal.dm [hunk 1]
```diff
@@ -101,6 +105,10 @@
 		real_name = name
 	if(!loc)
 		stack_trace("Simple animal being instantiated in nullspace")
+	NPC_count += 1
+	if(NPC_count > MAX_NPCs)
+		qdel(src)
+		visible_message("<span class='danger'>Die, monster! You don't belong in this world!</span>")
```
Comment (rhicora) [inline code/modules/mob/living/simple_animal/simple_animal.dm]: Does this need to be a visible message to everyone? It's kind of an OOC deletion.

**Comment by rhicora** [comment]
A global NPC cap is a very good change but I would make the message an admin log instead of a public-visible message.

File: code/modules/mob/living/simple_animal/simple_animal.dm (hunk 1 above)
Comment (UndeadNappist) [reply code/modules/mob/living/simple_animal/simple_animal.dm]: Only admins get meme humor now.

**Comment by UndeadNappist** [comment]
Only admins are able to see the meme now.

**Pull request merged**